src/OrchardCore.Modules/OrchardCore.Themes/Views/Admin/Index.cshtml

```diff
@@ -59,7 +59,7 @@ else
                         }
                         else
                         {
-                            @T["This is the default {0} theme", themeEntry.IsAdmin ? "Admin" : "Site"]
+                            @T["This is the default {0} theme", themeEntry.IsAdmin ? @T["Admin"] : @T["Site"]]
```

Comment:
If a parameter needs to be translated, then it should use the `IStringLocalizer` instead.
Another simpler option is to use two strings: 
```suggestion
                            if (themeEntry.IsAdmin)
                            {
                                @T["This is the default Admin theme"]
                            }
                            else
                            {
                                @T["This is the default Site theme"]
                            }
```